### PR TITLE
test_integration.py bug

### PR DESCRIPTION
**Pull request by dantavori**
## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/20372

## Description
Fixed the issue in `test_integration.py` in which whenever an investigation experienced an error during the CircleCI build, the body of the error didn't seep to the logs.

## Screenshots
image

## Does it break backward compatibility?
   - No

## Additional information
Currently, there's also a change in Hybrid Analysis integration that causes it to fail in order to see the fix in the build.

### CODE REVIEW

File: Integrations/integration-HybridAnalysis.yml [hunk 1]
```diff
@@ -299,7 +299,7 @@ script:
                     }
                 }
             }
-        var res = sendRequest('POST', '/api/v2/search/terms', body);
+        var res = sendRequest('POST', '/api/v2/search/termssssss', body);
```
Comment (yuvalbenshalom) [inline Integrations/integration-HybridAnalysis.yml]: `termssssss` is this for real?

Comment (dantavori) [reply Integrations/integration-HybridAnalysis.yml]: read the additional information...

File: Tests/test_integration.py [hunk 2]
```diff
@@ -371,10 +371,10 @@ def __print_investigation_error(client, playbook_id, investigation_id, color=LOG
...
         print_color('Playbook ' + playbook_id + ' has failed:', color)
         for entry in entries:
-            if entry['type'] == ENTRY_TYPE_ERROR:
-                if entry['parentContent']:
-                    print_color('\t- Command: ' + entry['parentContent'].encode('utf-8'), color)
-                print_color('\t- Body: ' + entry['contents'].encode('utf-8'), color)
+            if entry['type'] == ENTRY_TYPE_ERROR and entry['parentContent']:
+                print_color('- Task ID: ' + entry['taskId'].encode('utf-8'), color)
+                print_color('  Command: ' + entry['parentContent'].encode('utf-8'), color)
+                print_color('  Body:\n' + entry['contents'].encode('utf-8') + '\n', color)
```
Comment (ronykoz) [inline Tests/test_integration.py]: why did we merge this into one if?

File: Integrations/integration-HybridAnalysis.yml (hunk 1 above)
Comment (ronykoz) [reply Integrations/integration-HybridAnalysis.yml]: revert once we review

**Comment by ronykoz** [comment]
@dantavori lets go over this and merge this ASAP

File: Tests/test_integration.py (hunk 2 above)
Comment (dantavori) [reply Tests/test_integration.py]: If I'm not mistaken, entries without `parentContent` are like this one:
image
which doesn't carry useful information - if you think it's relevant, I'll revert it

File: Integrations/integration-HybridAnalysis.yml (hunk 1 above)
Comment (ronykoz) [reply Integrations/integration-HybridAnalysis.yml]: lets revert and merge this

**Comment by ronykoz** [comment]
Merging due to sample tests

**Pull request merged**